packages/pilot/src/containers/RecipientDetails/Balance/index.js

```diff
@@ -0,0 +1,550 @@
+                  <Operations
+                    columns={translateColumns(getColumns(typesLabels))}
+                    currentPage={currentPage}
+                    dates={dates}
+                    disabled={disabled}
+                    emptyMessage={t('models.operations.empty_message')}
+                    exportLabel={t('models.operations.export')}
+                    exporting={exporting}
+                    itemsPerPage={itemsPerPage}
+                    loading={disabled || loading}
+                    labels={{
+                      empty: t('models.operations.empty_message'),
+                      exportCall: t('export_table'),
+                      exportTo: t('export_to'),
+                      results: t(
+                        'pages.balance.results',
+                        { count: operations.total }
+                      ),
+                      totalOf: t('pages.balance.total.of'),
+                    }}
+                    onPageChange={this.handleOperationsPageChange}
+                    onPageCountChange={onPageCountChange}
+                    onExport={onExport}
+                    pageSizeOptions={pageSizeOptions}
+                    rows={operations.rows}
+                    t={t}
+                    title={t('pages.balance.operations_title')}
+                    totalPages={operations.count}
+                  />
+                </CardContent>
+              </CardSection>
+            </Col>
+          </Row>
+        </Grid>
+      </CardContent>
+    )
+  }
+}
+
+const cashFlowShape = PropTypes.arrayOf(PropTypes.shape({
+  amount: PropTypes.number.isRequired,
+  type: PropTypes.string.isRequired,
+})).isRequired
+
+const numberOrStringShape = PropTypes.oneOfType([
+  PropTypes.string.isRequired,
+  PropTypes.number.isRequired,
+]).isRequired
+
+RecipientBalance.propTypes = {
+  anticipation: PropTypes.shape({
+    available: PropTypes.number,
+    error: PropTypes.bool.isRequired,
+    loading: PropTypes.bool.isRequired,
+  }).isRequired,
+  balance: PropTypes.shape({
+    amount: PropTypes.number.isRequired,
+    available: PropTypes.shape({
+      withdrawal: PropTypes.number.isRequired,
+    }),
+    outcoming: PropTypes.number.isRequired,
+  }).isRequired,
+  currentPage: PropTypes.number.isRequired,
+  dates: PropTypes.shape({
+    end: PropTypes.instanceOf(moment),
+    start: PropTypes.instanceOf(moment),
+  }).isRequired,
+  disabled: PropTypes.bool.isRequired,
+  exporting: PropTypes.bool.isRequired,
+  itemsPerPage: PropTypes.number.isRequired,
+  loading: PropTypes.bool.isRequired,
+  onAnticipationClick: PropTypes.func.isRequired,
+  onCancelRequestClick: PropTypes.func,
+  onExport: PropTypes.func.isRequired,
+  onFilterClick: PropTypes.func.isRequired,
+  onPageChange: PropTypes.func.isRequired,
+  onPageCountChange: PropTypes.func.isRequired,
+  onWithdrawClick: PropTypes.func.isRequired,
+  pageSizeOptions: PropTypes.arrayOf(PropTypes.number),
```

Comment:
Essa prop é uma lista de objetos com `name` e `value`, o que está causando erros no console. Esse propTypes deveria ser:

```suggestion
  pageSizeOptions: PropTypes.arrayOf(PropTypes.shape({
    name: PropTypes.string,
    value: PropTypes.string,
  })).isRequired,
```